src/scenes/Event/components/progress.js

```diff
@@ -0,0 +1,11 @@
+import styled, { css } from 'styled-components';
+import { LinearProgress } from 'material-ui/Progress';
+
+
+export const Progress = styled(LinearProgress)`
```

Comment:
is this a new way of defining styles? check with @liviavinci on this one.

Reply:
@alex-cory please use a styles.js for this.